CHANGES.md

```diff
@@ -47,7 +47,7 @@ creating a new release entry be sure to copy & paste the span tag with the
 `actions:bind` attribute, which is used by a regex to find the text to be
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
-## __cylc-8.0b0 (<span actions:bind='release-date'>2020-08?</span>)__
+## __cylc-8.0b0 (<span actions:bind='release-date'>Released 2021-03-26</span>)__
```

Comment:
Unimportant but this may be outdated by release.

Reply:
I've pushed a tweak to the date.